examples/simple/src/comments/PostQuickCreate.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import {
+    CREATE,
+    FETCH_START,
+    FETCH_END,
+    LongTextInput,
+    SimpleForm,
+    TextInput,
+    Toolbar,
+    required,
+} from 'react-admin'; // eslint-disable-line import/no-unresolved
+import dataProvider from '../dataProvider';
+import CancelButton from './PostQuickCreateCancelButton';
+import SaveButton from './PostQuickCreateSaveButton';
+
+const PostQuickCreateToolbar = ({ submitting, onCancel, ...props }) => (
```

Comment:
Why do you need a custom toolbar? To close the window on cancel? Then add a comment to explain it, it's not immediate.